src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -1359,10 +1359,6 @@ void EnBigpo_DrawScoopSoul(Actor* thisx, GlobalContext* globalCtx) {
     CLOSE_DISPS(globalCtx->state.gfxCtx);
 }
 
-/*
- * this matches without OPENDISPS but with it has stack issues,
- *  might be able to find an alternative match with the macros, so far no success
- */
```

Comment:
So... what changed? I don't see anything special in the diff

Reply:
To be honest I have no idea. I was able to just use Macros with no issues.